esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -1327,9 +1327,9 @@ def get_student_schedules(request, students, prog, extra='', onsite=False):
             return render_to_response(basedir+'studentschedule.html', request, context)
         elif file_type == 'pdf':
             if len(students) > 1:
-                response = HttpResponse(content_type='application/pdf')
+                response = HttpResponse(content=render_to_latex(basedir+'studentschedule.tex', context, 'pdf'),
+                                        content_type='application/pdf')
                 response['Content-Disposition'] = 'attachment; filename="studentschedules.pdf"'
-                response.write(render_to_latex(basedir+'studentschedule.tex', context, 'pdf'))
```

Comment:
@willgearty what is this supposed to do? I'm not sure I understand

Reply:
Not sure why, but on Python 3 the student schedule download was broken. This fixes it.

Reply:
can you help me understand what this change does? This is the file that causes the database errors that are making the PR tests fail

Reply:
That's bizarre. There shouldn't be anything different here than what was happening before on Python 2. We render the schedules as a pdf, then set the response as a file download. That's basically it.

Reply:
yeah I don't understand it. Maybe best to discuss in real time.... Or maybe this was actually mitigating an error in another file.  Stack seems to indicate these requests are notoriously asynchronous/hard to debug 🤷 😢